jvm-packages/xgboost4j-spark/src/test/scala/ml/dmlc/xgboost4j/scala/spark/Utils.scala

```diff
@@ -16,15 +16,6 @@
 
 package ml.dmlc.xgboost4j.scala.spark
 
-import java.io.File
-
 trait Utils extends Serializable {
```

Comment:
currently, SharedContext also contains cleanExternalCache which is also unrelated....so if we really want to make it absolutely consistent, we can rename SharedContext to something else